### PR TITLE
fix after kafka steaming | vcnt | ecnt |

### PR DESCRIPTION
**Pull request by thamindumk**

### CODE REVIEW

**Review by miyurud — APPROVED** [review]
LGTM

File: src/partitioner/stream/Partitioner.cpp [hunk 1]
```diff
@@ -133,6 +133,26 @@ void Partitioner::printStats() {
...
+
+    for (auto partition : this->partitions) {
+        vertexCount += partition.getVertextCount();
+        edgesCount += partition.getEdgesCount();
+        edgeCutsCount += partition.edgeCutsCount();
+    }
+
+    double numberOfEdges = edgesCount + edgeCutsCount/2;
+    std::string sqlStatement = "UPDATE graph SET vertexcount = '" + std::to_string(vertexCount) +
+                          "' ,centralpartitioncount = '" + std::to_string(this->numberOfPartitions) + "' ,edgecount = '" +
```
Comment (miyurud) [inline src/partitioner/stream/Partitioner.cpp:149]: Can you break this line into two? We need to ensure the line length of less than 120 characters.

File: src/util/kafka/StreamHandler.h [hunk 2]
```diff
@@ -23,7 +23,7 @@ limitations under the License.
 
 class StreamHandler {
  public:
-    StreamHandler(KafkaConnector *kstream, int numberOfPartitions, std::vector<DataPublisher *> &workerClients);
+    StreamHandler(KafkaConnector *kstream, int numberOfPartitions, std::vector<DataPublisher *> &workerClients, SQLiteDBInterface* sqlite);
```
Comment (miyurud) [inline src/util/kafka/StreamHandler.h:26]: Can you break this line into two? We need to ensure the line length of less than 120 characters.

**Review by miyurud — CHANGES_REQUESTED** [review]
Please address the two new comments I have put and reply how you addressed them.

File: src/partitioner/stream/Partitioner.cpp (hunk 1 above)
Comment (thamindumk) [reply src/partitioner/stream/Partitioner.cpp]: I broke this line

File: src/util/kafka/StreamHandler.h (hunk 2 above)
Comment (thamindumk) [reply src/util/kafka/StreamHandler.h]: I broke this line

File: src/partitioner/stream/Partitioner.cpp (hunk 1 above)
Comment (miyurud) [reply src/partitioner/stream/Partitioner.cpp]: Ok

File: src/util/kafka/StreamHandler.h (hunk 2 above)
Comment (miyurud) [reply src/util/kafka/StreamHandler.h]: Ok

**Pull request closed without merging**